as-vending/config/configuration.go

```diff
@@ -0,0 +1,119 @@
+package config
+
+import (
+	"fmt"
+)
+
+type ServiceConfig struct {
+	AppCustom AppCustomConfig
```

Comment:
AppCustiom was just a sample name for the example. Name needs to be specific in actual implementations

```suggestion
	Vending VendingConfig
```